scratch-debugger/debug.sh

```diff
@@ -113,12 +114,16 @@ case ${ARCH} in
     exit 1
 esac
 
-DOCKERCMD="/mnt/rootfs/usr/bin/docker -H unix:///run/docker.sock"
+DOCKERCMD="/tmp/docker/docker -H unix:///mnt/docker.sock"
 
 # Command for installing busybox image from the debugger container into the target container.
 INSTALLCMD="set -x;" # Print commands, for debugging.
+# Download docker client
+INSTALLCMD="${INSTALLCMD} wget -qO/tmp/docker.tgz ${DOCKER_DOWNLOAD_URL}"
+# Extract docker client
+INSTALLCMD="${INSTALLCMD} && tar zxvf /tmp/docker.tgz -C /tmp"
 # Create the directory structure for the install.
-INSTALLCMD="${INSTALLCMD} mkdir -p ${INSTALL_DIR}"
+INSTALLCMD="${INSTALLCMD} && mkdir -p ${INSTALL_DIR}"
 # Copy the directory structure into the target container.
 INSTALLCMD="${INSTALLCMD} && ${DOCKERCMD} cp /tmp ${CONTAINER_ID}:/"
```

Comment:
This line needs to be changed, because we don't want to copy the downloaded docker into the target container. It was wrong before - it assumed `INSTALL_DIR` was under `/tmp`.

Reply:
oh I didn't realize the entire `/tmp` got copied there. I will wget docker into `/docker-cli`.

Reply:
Yeah, it's a hack since docker doesn't have a `mkdir` equivalent, nor can you copy into a directory that doesn't exist in the target.